layouts/page/pulumi-up.html

```diff
@@ -1,39 +1,105 @@
+                                <p class="max-w-3xl mx-auto text-lg">{{ .Params.peers.description | markdownify }}</p>
+                                <div class="mt-12 text-center">
+                                    <a href="#register" class="btn btn-lg btn-orange">Grab Your Spot</a>
+                                </div>
+                            </div>
+                        </div>
                     </div>
                 </div>
-                <div class="text-center p-4">
-                    <h4 class="text-gray-100 md:mr-8 mt-2 text-2xl">{{ .Params.form.label }}</h4>
-                    <div class="flex items-center justify-center">
-                        <pulumi-hubspot-form form-id="{{ .Params.form.form_id }}" class="newsletter newsletter-wide"></pulumi-hubspot-form>
+            </section>
+            <section id="sessions" class="mt-16">
+                <div class="flex flex-col lg:flex-row">
+                    <div class="w-full lg:w-1/2 p-4">
+                        <div data-title="Workshops" class="neon-callout-box-orange text-center">
+                            <p class="coming-soon-sign">Coming Soon</p>
+                        </div>
                     </div>
-                </div>
-                <div class="w-full flex items-center justify-center mt-8 p-4">
-                    <div class="relative inline-block">
-                        <p class="mb-12 text-2xl">{{ .Params.quote.quote }}</p>
-                        <p class="absolute bottom-0 right-0 text-lg italic">~ {{ .Params.quote.name }}</p>
+
+                    <div class="w-full lg:w-1/2 p-4">
+                        <div data-title="Binge Content" class="neon-callout-box-blue text-center">
+                            <p class="coming-soon-sign">Coming Soon</p>
+                        </div>
                     </div>
                 </div>
             </section>
+            <section id="register" class="mb-0 py-16 text-white px-4 lg:px-0">
+                <div class="container mx-auto w-full lg:w-1/2 bg-white py-8 px-10 text-sm rounded rounded-lg shadow-lg border border-gray-300 relative">
+                    {{ with .Params.register_form }}
+                        <div class="mx-auto">
+                            <h2 class="text-black text-center">{{ .headline }}</h2>
+                            <div class="hs-form hs-form-fg-light hs-demo-form text-black">
+                                <pulumi-hubspot-form form-id="{{ .hubspot_form_id }}"></pulumi-hubspot-form>
+                            </div>
+                        </div>
+                    {{ end }}
+                </div>
+            </section>
         </div>
     </div>
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Vibur" />
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Walter+Turncoat" />
```

Comment:
Where is this used?

Reply:
Ah, good catch! This was leftover when I was experimenting with a chalkboard feel.